app/config/server.ts

```diff
@@ -65,6 +65,7 @@ export const getServerSideConfig = () => {
   }
 
   const isAzure = !!process.env.AZURE_URL;
+  const isGoogle = !!process.env.GOOGLE_URL;
```

Comment:
User may using google models by only setting the GOOGLE_API_KEY environment variable. Maybe you should switching to the environment variable to check whether users expecting to use google model